adafruit_pioasm.py

```diff
@@ -60,6 +60,8 @@ def assemble(text_program):
     assembled = []
     for instruction in instructions:
         print(instruction)
+        if len(instruction) == 0:
+            continue
```

Comment:
I think we want this on line 41. The comment shouldn't end up in instructions because it will mess up the label location (line 55).

Reply:
Sorry, I don't use JUMP yet in PIO... so I would not notice and I don't have a test case to validate or invalidate the result.
Feel free to fix it another way.

While for most library, it is hard to have regression test because it is hardware, the assembler hardware independent, so it is a perfect target for automated testing with known input and output.